Documentation/compatibility/wpf-focus-visual-for-radiobutton-and-checkbox-displays-correctly-when-there-is-no-content.md

```diff
@@ -0,0 +1,47 @@
+## WPF FocusVisual for RadioButton and CheckBox Now Displays Correctly When The Controls Have No Content
+
+### Scope
+Edge
+
+### Version Introduced
+4.7.2
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+In the .NET Framework 4.7.1 and earlier versions, WPF `System.Windows.Controls.CheckBox` and `System.Windows.Controls.RadioButton` have inconsistent and, in Classic and High Contrast themes,
+incorrect FocusVisuals.  These issues occur in cases where the controls do not have any content set.  This can make the transition between themes confusing and the FocusVisual hard to see.
+In the .NET Framework 4.7.2, these visuals are now more consistent across themes and more easily visible in Classic and High Contrast themes.
+
+- [X] Quirked
+- [ ] Build-time break
+
+### Recommended Action
+A developer who wants to utilize this change while targeting a framework version below .NET 4.7.2 or a developer who requires the previous functionality while targeting .NET 4.7.2 or greater 
+can set the following set of AppContext flags appropriately.
+
+Note that all the flags must be set appropriately.  WPF applications are required to opt in to all earlier accessibility improvements to get the later improvements. 
+To do this, ensure that if the AppContext switch 'Switch.UseLegacyAccessibilityFeatures.N' is set to 'false', then 'Switch.UseLegacyAccessibilityFeatures' and all 
+'Switch.UseLegacyAccessibilityFeatures.M' switches, when M < N, evaluate to false as well. Note that, if a switch for a particular set of accessibility improvements is not present, 
```

Comment:
Since this is the first numbered switch, I don't think that it's necessary -- it only will be for the next version, when there will be Switch.2 and Switch.3, presumably. So we only need to mention that, "WPF applications must opt in to all early accessibility improvements to get the later improvements. In other words, both the \`Switch.UseLegacyAccessibilityFeatures\` and the \`Switch.UseLegacyAccessibilityFeatures.2\` switches must be set."

Reply:
Fair enough, we use the same language in the localized error that's inside the products (it's the same for WinForms), so I wanted to maintain consistency with that.  But we can pair it down if that makes more sense.